### PR TITLE
Update free-programming-books-langs.md

### PR DESCRIPTION
**Pull request by park-young-chun**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [ ] Search for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by park-young-chun** [comment]
Update free-programming-books-langs.md

Java Masters Web Page Link Update!
- OOPs Concepts in Java
- Exception Handling
- Java Collection
- Framework
- Java Concurrency
- Java Web Programming
- Spring

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1013,6 +1013,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Java Application Development on Linux (2005)](https://ptgmedia.pearsoncmg.com/images/013143697X/downloads/013143697X_book.pdf) - Carl Albing and Michael Schwarz (PDF)
 * [Java, Java, Java Object-Oriented Problem Solving](https://archive.org/details/JavaJavaJavaObject-orientedProblemSolving/page/n0) - R. Morelli and R.Walde
 * [Java Language and Virtual Machine Specifications](https://docs.oracle.com/javase/specs/) - James Gosling, et al
+* [Java Masters](https://javamasters.io/servlets) - Java Masters (Web Page)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1016]: usually we would denote theses as `(HTML)` rather than `(Web Page)`

**Comment by park-young-chun** [comment]
Update free-programming-books-langs.md
Web Page -> HTML

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**